src/browser/modules/D3Visualization/lib/visualization/components/Visualization.ts

```diff
@@ -278,18 +278,40 @@ const vizFn = function (
     }
 
     function dragHandler(simulation: Simulation<VizNode, Relationship>) {
+      let initialPos: [number, number]
+      let restartedSim = false
+      const tolerance = 25
+
+      const dist = (a: [number, number], b: [number, number]) =>
+        Math.pow(a[0] - b[0], 2) + Math.pow(a[1] - b[1], 2)
+
       function dragstarted(event: D3DragEvent<SVGGElement, VizNode, any>) {
-        clearSimulationTimeout()
-        if (!event.active) simulation.alphaTarget(0.03).alpha(1).restart()
+        initialPos = [event.x, event.y]
+        restartedSim = false
       }
 
-      function dragged(event: D3DragEvent<SVGGElement, VizNode, any>) {
-        event.subject.fx = event.x
-        event.subject.fy = event.y
+      function dragged(
+        event: D3DragEvent<SVGGElement, VizNode, any>,
+        node: VizNode
+      ) {
+        // This is to prevent clicks/double clicks from restarting the simulation
+        if (dist(initialPos, [event.x, event.y]) > tolerance && !restartedSim) {
+          // Set alphaTarget to a value higher than alphaMin so the simulation
+          // isn't stopped while nodes are being dragged.
+          simulation.alphaTarget(0.09).alpha(1).restart()
```

Comment:
If you double click on a node or do a super short drag, the wiggle of surrounding nodes is still a bit jarring imo. I not necessarily a problem, but maybe setting the alpha to a lower value when dragging would make it better (?) (bit out of my depths on the simulation/target/alpha stuff tbh)  

https://user-images.githubusercontent.com/10564538/150495328-01f9e362-87d0-427c-a955-e8792fbbfa75.mp4

Reply:
It'd be nice if "unrelated" nodes moved less when dragging as well, perhaps solved by the same change